src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int DATA_TYPE_INDEX = 0;
+    private static final int DATA_VALUE_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileData = readFromFile(fromFileName);
+        String report = reportCreation(fileData);
```

Comment:
```suggestion
        String report = createReport(fileData);
```
methods names should follow this pattern `doSomething()`